packages/react-core/src/components/Alert/examples/AlertGroupAnimations.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
```

Comment:
Since `hasAnimations` is true by default.  I would say this examples is not technically needed correct?  I wonder if we can combine the docs from this with another example.  @edonehoo I would be interested in your thoughts here.

Reply:
Hmm I would agree that this example doesn't seem necessary if hasAnimations is the default behavior. 

Or I'm wondering what we're trying to explain via this example that we aren't doing in previous examples? Is it this part? _"When using animations, each alert must have a unique id or key passed to it."_  

We could add an asterisk/update the description of id in the props table saying that it's required when using animations?